priiloader/source/main.cpp

```diff
@@ -2861,7 +2861,19 @@ int main(int argc, char **argv)
 	AddMem2Area (14*1024*1024, OTHER_AREA);
 	LoadHBCStub();
 	gprintf("\"Magic Priiloader word\": %x - %x",*(vu32*)MAGIC_WORD_ADDRESS_2 ,*(vu32*)MAGIC_WORD_ADDRESS_1);
-	LoadSettings();
+
+	bool IsFirstTimeUse = false;

```

Comment:
`bool IsFirstTimeUse = LoadSettings() == LOADSETTINGS_INI_CREATED;`
no need for the complete switch imo. and even if you'd keep the switch/case i'd make it
```	switch (LoadSettings())
	{
		case LOADSETTINGS_INI_CREATED:
			IsFirstTimeUse = true;
		case LOADSETTINGS_OK:
		case LOADSETTINGS_FAIL:
			// Do nothing, errors are handled later
			break;
	}```